src/main.c

```diff
@@ -999,6 +999,9 @@ struct tracing_params{
 #ifdef USE_BDOPS_SUBMIT_BIO
 struct tracing_ops {
 	struct block_device_operations *bd_ops;
+#ifdef HAVE_BD_HAS_SUBMIT_BIO
+	bool has_submit_bio; // kernel version >= 6.4
+#endif
```

Comment:
As an option, for better readability, I'd propose to move a comment to the line above.
```suggestion
#ifdef HAVE_BD_HAS_SUBMIT_BIO
	// kernel version >= 6.4
	bool has_submit_bio;
#endif
```